docs/content/Schema/Reference/cube.mdx

```diff
@@ -631,6 +634,28 @@ cubes:
 It'll generate aliases for members such as `ofacts__count`. `sqlAlias` affects
 all member names including pre-aggregation table names.
 
+### <--{"id" : "Parameters"}--> sql_table
+
+The `sql_table` property can be used instead of [`sql`](#parameters-sql) to
+directly specify the name of a table from the data source that will be queried
+by this cube.
```

Comment:
```suggestion
The `sql_table` property is used as a concise way for defining a cube that uses a query like this: `SELECT * FROM table`. Instead of using the [`sql`](#parameters-sql) property, use `sql_table` with the table name that this cube will query.
```